xysched/README.md

```diff
@@ -32,11 +37,15 @@ Visit [pkg.go.dev](https://pkg.go.dev/github.com/xybor/xyplatform/xysched) for
 more details.
 
 # Example
-1. Print a message after one second.
+
+1.  Print a message after one second.
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/xybor/xyplatform/pullRequest?prid=10050498)
```suggestion
1. Print a message after one second.
```